tests/utils/index.ts

```diff
@@ -26,7 +28,64 @@ export const ARGS: TestingArguments = {
   cliVersion: process.env.PLUGINS_TEST_CLI_VERSION,
   cliPath: process.env.PLUGINS_TEST_CLI_PATH,
   linterVersion: parseLinterVersion(process.env.PLUGINS_TEST_LINTER_VERSION ?? ""),
+  dumpNewSnapshot: process.env.PLUGINS_TEST_NEW_SNAPSHOT?.toLowerCase() === "true",
```

Comment:
```suggestion
  dumpNewSnapshot: process.env.PLUGINS_TEST_NEW_SNAPSHOT?.toLowerCase() === "true",
```

just test if the envvar is set to something that isn't false-y - setting `ENV=1` is a common way to activate a given envvar